lib/utils/processPages.js

```diff
@@ -26,18 +28,24 @@ async function processPages (html, opts, savedApps, mappings) {
 }
 
 async function processFullDetailApps (apps, opts) {
-  const promises = apps.map(app => (
-    appDetails({
-      appId: app.appId,
-      lang: opts.lang,
-      country: opts.country,
-      cache: opts.cache,
-      throttle: opts.throttle,
-      requestOptions: opts.requestOptions
-    })
-  ));
-
-  return Promise.all(promises);
+  const getAppDetails = (app) => appDetails({
+    appId: app.appId,
+    lang: opts.lang,
+    country: opts.country,
+    cache: opts.cache,
+    throttle: opts.throttle,
+    requestOptions: opts.requestOptions
+  });
+
+  if (opts.throttle !== undefined) {
+    const throttle = pThrottle({
```

Comment:
why did you need to add this throttle setup? doesn't the setup in the internal request module apply here as well by just passing the throttle option?

Reply:
I thougth the same, but test fails if i don't edit processpage when it batch request fulldetails

Reply:
My bad, i reverted changes on this files, it now works perfectly, i'm on it.